requirements.txt

```diff
@@ -7,7 +7,6 @@ httpx
 kubernetes
 langchain
 langchain-community
-langchain-huggingface
```

Comment:
Looking at commit introducing this (https://github.com/opea-project/GenAIComps/commit/3e559df866972631e41a495b0bea068217224f86), Import / use of this module was added to `dataprep` and `retrievers` components, so you would need to add that module to their requirements files instead, otherwise this would break building of them.

Reply:
It's already there.

Reply:
Ah, sorry, those seems to be already using their own requirements, please set this as resolved:
```
$ git grep requirements.txt dataprep/ retrievers/ | grep -v md:
dataprep/src/Dockerfile:    pip install --no-cache-dir ${PIP_EXTRA_INDEX_URL} -r /home/user/comps/dataprep/src/requirements.txt && \
retrievers/src/Dockerfile:    pip install --no-cache-dir ${PIP_EXTRA_INDEX_URL} -r /home/user/comps/retrievers/src/requirements.txt && \
```

(Which makes me wonder why https://github.com/opea-project/GenAIComps/pull/1444 added that dep to here in the first place....)